tests/testsuite/config.rs

```diff
@@ -503,10 +503,7 @@ c = ['c']
     assert_error(
         config.get::<VSOB>("a").unwrap_err(),
         "\
-error in environment variable `CARGO_A`: could not load config key `a`
-
-Caused by:
-  invalid type: string \"x y\", expected a boolean or vector of strings",
+expected a boolean or vector of strings",
```

Comment:
oops, I think this happens because the `expecting = "..."` completely replaces the previous error. Not sure how to prevent this except to revert?